examples/api/test/cupertino/dialog/cupertino_alert_dialog.0_test.dart

```diff
@@ -22,4 +23,49 @@ void main() {
     await tester.pumpAndSettle();
     expect(find.text(actionText), findsNothing);
   });
+
+  testWidgets('Check for Directionality', (WidgetTester tester) async {
```

Comment:
Instead of in the example directory, can you move the test to `packages/flutter/test/cupertino/dialog_test.dart`?